Cargo.lock

```diff
@@ -681,6 +682,18 @@ dependencies = [
  "memoffset",
 ]
 
+[[package]]
+name = "nix"
+version = "0.26.2"
```

Comment:
Mhh, the entry before this one is about nix 0.23.1 (transitive dependency). Can we do this with nix 0.23 so that we don't pull in the dependency twice??

Reply:
Works for me to do that as a follow up then, no need to block this on it